packages/nouns-contracts/contracts/governance/NounsDAOLogicV2.sol

```diff
@@ -0,0 +1,895 @@
+        require(msg.sender == admin, 'NounsDAO::_setProposalThresholdBPS: admin only');
+        require(
+            newProposalThresholdBPS >= MIN_PROPOSAL_THRESHOLD_BPS &&
+                newProposalThresholdBPS <= MAX_PROPOSAL_THRESHOLD_BPS,
+            'NounsDAO::_setProposalThreshold: invalid proposal threshold'
+        );
+        uint256 oldProposalThresholdBPS = proposalThresholdBPS;
+        proposalThresholdBPS = newProposalThresholdBPS;
+
+        emit ProposalThresholdBPSSet(oldProposalThresholdBPS, proposalThresholdBPS);
+    }
+
+    function _setMinQuorumVotesBPS(uint16 newMinQuorumVotesBPS) external {
+        require(msg.sender == admin, 'NounsDAO::_setMinQuorumVotesBPS: admin only');
+        DynamicQuorumParams memory params = getDynamicQuorumParamsAt(block.number);
+
+        require(
+            newMinQuorumVotesBPS >= MIN_QUORUM_VOTES_BPS_LOWER_BOUND &&
+                newMinQuorumVotesBPS <= MIN_QUORUM_VOTES_BPS_UPPER_BOUND,
+            'NounsDAO::_setMinQuorumVotesBPS: invalid min quorum votes bps'
+        );
+        require(
+            newMinQuorumVotesBPS <= params.maxQuorumVotesBPS,
+            'NounsDAO::_setMinQuorumVotesBPS: min quorum votes bps greater than max'
+        );
+
+        uint16 oldMinQuorumVotesBPS = params.minQuorumVotesBPS;
+        params.minQuorumVotesBPS = newMinQuorumVotesBPS;
+
+        _writeQuorumParamsCheckpoint(params);
+
+        emit MinQuorumVotesBPSSet(oldMinQuorumVotesBPS, newMinQuorumVotesBPS);
+    }
+
+    function _setMaxQuorumVotesBPS(uint16 newMaxQuorumVotesBPS) external {
+        require(msg.sender == admin, 'NounsDAO::_setMaxQuorumVotesBPS: admin only');
+        DynamicQuorumParams memory params = getDynamicQuorumParamsAt(block.number);
+
+        require(
+            newMaxQuorumVotesBPS <= MAX_QUORUM_VOTES_BPS_UPPER_BOUND,
+            'NounsDAO::_setMaxQuorumVotesBPS: invalid max quorum votes bps'
+        );
+        require(
+            params.minQuorumVotesBPS <= newMaxQuorumVotesBPS,
+            'NounsDAO::_setMaxQuorumVotesBPS: min quorum votes bps greater than max'
+        );
+
+        uint16 oldMaxQuorumVotesBPS = params.maxQuorumVotesBPS;
+        params.maxQuorumVotesBPS = newMaxQuorumVotesBPS;
+
+        _writeQuorumParamsCheckpoint(params);
+
+        emit MaxQuorumVotesBPSSet(oldMaxQuorumVotesBPS, newMaxQuorumVotesBPS);
+    }
+
+    function _setQuorumVotesBPSOffset(uint16 newQuorumVotesBPSOffset) external {
+        require(msg.sender == admin, 'NounsDAO::_setQuorumVotesBPSOffset: admin only');
+        DynamicQuorumParams memory params = getDynamicQuorumParamsAt(block.number);
+
+        uint16 oldQuorumVotesBPSOffset = params.quorumVotesBPSOffset;
+        params.quorumVotesBPSOffset = newQuorumVotesBPSOffset;
+
+        _writeQuorumParamsCheckpoint(params);
+
+        emit QuorumVotesBPSOffsetSet(oldQuorumVotesBPSOffset, newQuorumVotesBPSOffset);
+    }
+
+    function _setQuorumLinearCoef(uint32 newQuorumLinearCoef) external {
```

Comment:
Nit: Do not abbreviate `coefficient`

Reply:
fixed https://github.com/nounsDAO/nouns-monorepo/pull/438/commits/5822d9fec5966061fd45e45f1a0c180466c2173d